addons/purchase/models/product.py

```diff
@@ -60,6 +61,11 @@ class ProductProduct(models.Model):
 
     purchased_product_qty = fields.Float(compute='_compute_purchased_product_qty', string='Purchased',
         digits='Product Unit of Measure')
+    
+    product_catalog_product_is_in_purchase_order = fields.Boolean(
```

Comment:
would go for a simpler naming as product_is_in_purchase_order

Reply:
Isn't it better to be consistent with the [one on sales order](https://github.com/odoo/odoo/blob/17.0/addons/sale/models/product_product.py#L16-L19)?

Reply:
It could be useful outside catalog. The naming in sale order is not super. I prefer to have something explicit